File: src/main/webapp/osca/common/domain/Referentiel/Rubriques/SousRubriques/Dev/FeuilleDeRouteDev.ts
import { CriteresDev } from './CriteresDev';
import { PratiquesDev } from './PratiquesDev';

export interface FeuilleDeRouteDev {
    titre: string;
    keyFeuilleDeRouteDev: string;
    criteresDevList: CriteresDev[];
    pratiquesDevList: PratiquesDev[];
}

File: src/main/java/com/soprasteria/osca/domain/referentiel/rubriques/sousrubriques/dev/FeuilleDeRouteDev.java
package com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.dev;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class FeuilleDeRouteDev {

    private String titre;
    private String keyFeuilleDeRouteDev;
    private List<CriteresDev> criteresDevList;
    private List<PratiquesDev> pratiquesDevList;
}
